app/basicComponents/Input.tsx

```diff
@@ -146,11 +146,12 @@ const Input = ({
   min,
   max,
 }: Props) => {
-  let debounce: any = null;
   const [isFocused, setIsFocused] = useState(false);
+  const [debounce, setDebounce] = useState<NodeJS.Timeout | null>(null);
```

Comment:
`NodeJS.Timeout` — this is the wrong type actually, since it runs in the browser environment, it is just a number ;)